package.json

```diff
@@ -1,74 +1,96 @@
 {
-  "name": "modelbrouwers",
-  "version": "1.3.12",
-  "description": "Bundle for Modelbrouwers.nl community website",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/modelbrouwers/modelbrouwers.git"
-  },
-  "keywords": [
-    "phpbb",
-    "django",
-    "forum"
-  ],
-  "author": "Sergei Maertens <sergeimaertens@gmail.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/modelbrouwers/modelbrouwers/issues"
-  },
-  "homepage": "https://github.com/modelbrouwers/modelbrouwers",
-  "dependencies": {
-    "bootstrap": "^3.3.7",
-    "bourbon": "^5.0.0",
-    "fine-uploader": "~5.1.3",
-    "font-awesome": "^4.7.0",
-    "gulp": "^3.9.1",
-    "jquery": "^3.3.1"
-  },
-  "devDependencies": {
-    "bourbon-neat": "^1.8.0",
-    "compass-importer": "^0.4.1",
-    "gulp-autoprefixer": "^5.0.0",
-    "gulp-jshint": "^2.1.0",
-    "gulp-sass": "^4.0.1",
-    "gulp-sourcemaps": "^2.6.4",
-    "jspm": "^0.16.14",
-    "node-sass": "^4.9.0",
-    "uglify-js": "^2.4.23"
-  },
-  "jspm": {
-    "directories": {
-      "baseURL": "static",
-      "doc": "doc"
+    "name": "modelbrouwers",
+    "version": "1.3.12",
+    "description": "Bundle for Modelbrouwers.nl community website",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "webpack --config webpack.config.js --mode=production",
```

Comment:
this seems to keep `webpack` in watch mode when I run `npm run build` - can you set it up so that `npm run dev` is in watch mode and `npm run build` does a one-time build? I need to be able to script this in ansible.